packages/core/admin/admin/src/translations/fr.json

```diff
@@ -526,5 +526,32 @@
   "request.error.model.unknown": "Le model n'existe pas",
   "selectButtonTitle": "Sélectionner",
   "skipToContent": "Aller au contenu",
-  "submit": "Soumettre"
+  "submit": "Soumettre",
+  "components.Blocks.modifiers.bold": "Gras",
+  "components.Blocks.modifiers.italic": "Italique",
+  "components.Blocks.modifiers.underline": "Souligné",
+  "components.Blocks.modifiers.strikethrough": "Barré",
+  "components.Blocks.modifiers.code": "Code",
+  "components.Blocks.link": "Lien",
+  "components.Blocks.expand": "Agrandir",
+  "components.Blocks.collapse": "Rétrécir",
+  "components.Blocks.popover.text": "Texte",
+  "components.Blocks.popover.text.placeholder": "Enter link text",
+  "components.Blocks.popover.link": "Lien",
+  "components.Blocks.popover.link.placeholder": "Coller un lien",
+  "components.Blocks.popover.link.error": "Please enter valid link",
```

Comment:
Should this still be in english?

Reply:
I missed this one, but I'm not sure we use it since the "Save" button is disabled until we paste/type a valid link. I translated it anyway, just in case 👍